shell/platform/fuchsia/flutter/tests/flatland_platform_view_unittest.cc

```diff
@@ -929,15 +931,43 @@ TEST_F(FlatlandPlatformViewTests, DestroyViewTest) {
         on_view_unbound(std::move(content_id));
       };
 
+  bool create_view_called = false;
+  auto on_create_view =
+      [&create_view_called, this](
+          int64_t view_id, flutter_runner::ViewCallback on_view_created,
+          flutter_runner::FlatlandViewCreatedCallback on_view_bound,
+          bool hit_testable, bool focusable) {
+        create_view_called = true;
+        on_view_created();
+        fuchsia::ui::composition::ContentId content_id;
+        on_view_bound(std::move(content_id), MakeChildViewWatcher());
+      };
+
   auto platform_view = PlatformViewBuilder(delegate, std::move(task_runners))
-                           .SetDestroyViewCallback(DestroyViewCallback)
+                           .SetCreateViewCallback(on_create_view)
+                           .SetDestroyViewCallback(on_destroy_view)
                            .Build();
 
   // Cast platform_view to its base view so we can have access to the public
   // "HandlePlatformMessage" function.
   auto base_view = static_cast<flutter::PlatformView*>(&platform_view);
   EXPECT_TRUE(base_view);
 
+  std::ostringstream create_message;
+  create_message << "{"
+                 << "    \"method\":\"View.create\","
+                 << "    \"args\": {"
+                 << "       \"viewId\":42,"
+                 << "       \"hitTestable\":true,"
+                 << "       \"focusable\":true"
+                 << "    }"
+                 << "}";
+
+  auto create_response = FakePlatformMessageResponse::Create();
+  base_view->HandlePlatformMessage(create_response->WithMessage(
+      "flutter/platform_views", create_message.str()));
+  RunLoopUntilIdle();
```

Comment:
What's the expectation here? That the code doesn't crash, right? There's not state to intentionally inspect after the runloop?

// View set up with no crash. 

Oh, there is that create_view_called bool.
ASSERT_TRUE(create_view_called) << "view set up with no crash";
?